gradlekotlinconverter.kts

```diff
@@ -141,7 +141,7 @@ fun String.convertDependencies(): String {
 // becomes
 // signingConfig = signingConfigs.getByName("release")
 fun String.convertCompileToImplementation(): String {
-    val outerExp = "(compile|testCompile).*\".*\"".toRegex()
+    val outerExp = "(compile|testCompile)[^O].*\".*\"".toRegex()
```

Comment:
Could you explain why the [^O]?

Reply:
Well, that was to say any character besides the letter "O", so not matching `compileOnly` and `testCompileOnly`.  As you questioned it though I've improved it to be (?!O) which means then only `testCompile` or `compile` if not followed by the letter "O" but the `(?!)` is a zero length negative lookahead, so it doesn't match anything.  Just refines the expression.

Reply:
Oh, nice find!